docs/manual/owb-to-pywb-deploy.rst

```diff
@@ -0,0 +1,65 @@
+Deploying pywb: Collection Paths and with Nginx/Apache
```

Comment:
Should this be "with" instead of "and with"?

Reply:
Thanks!